types/setting.go

```diff
@@ -930,6 +933,16 @@ var (
 		Default:  "1",
 	}
 
+	SettingDefinitionRecurringJobCreateSnapshotWhenRebuilding = SettingDefinition{
+		DisplayName: "Recurring Job Create Snapshot When Rebuilding",
+		Description: "This setting enables a recurring job to create a snapshot when replica rebuilding forcily .\n\n",
```

Comment:
Add the description for
- The potential risk if users enable the option.
- When the snapshot count reaches the maximum retain number and the option is not enabled, what will occur?

Reply:
Modified. And changed the setting to be `SettingDefinitionRecurringJobSnapshotDeleteFirst` and add the description for potential risk if this option is `false`.